quantum/rgblight.c

```diff
@@ -797,6 +805,16 @@ void rgblight_task(void) {
     // static light mode, do nothing here
     if ( 1 == 0 ) { //dummy
     }
+#ifdef RGBLIGHT_IDLE_ENABLE
+    // exit early if we are timedout
+    else if (rgblight_idle_timedout) {
+        return;
+    } else if (timer_elapsed32(rgblight_idle_timer) >= (RGBLIGHT_IDLE_TIMEOUT * 60000)) {
+        rgblight_idle_timedout = true;
+        rgblight_disable_noeeprom();
+        rgblight_idle_timer = timer_read32();
```

Comment:
does this do anything? maybe set it to 0?

Reply:
This was an oversight and is not needed - has been removed in the latest commit.